less/manual-bootstrap-overrides.less

```diff
@@ -0,0 +1,18 @@
+// After removing local bootstrap (and loading same version from cdn), this file fixes styles that broke
+
+// Variables needed for less compilation
```

Comment:
without these, compiling site.less breaks because it doesn't find the variables from bootstrap anymore